src/main/java/ca/on/oicr/gsi/dimsum/data/ProjectSummary.java

```diff
@@ -0,0 +1,339 @@
+
+    }
+
+    public void incrementFullDepthSeqCompletedCount() {
+      this.fullDepthSeqCompletedCount += 1;
+    }
+
+    // informatics review
+    public void informaticsPendingCount(int informaticsPendingCount) {
+      this.informaticsPendingCount = informaticsPendingCount;
+    }
+
+    public void informaticsCompletedCount(int informaticsCompletedCount) {
+      this.informaticsCompletedCount = informaticsCompletedCount;
+    }
+
+    // draft report
+    public void draftReportPendingCount(int draftReportPendingCount) {
+      this.draftReportPendingCount = draftReportPendingCount;
+    }
+
+    public void draftReportCompletedCount(int draftReportCompletedCount) {
+      this.draftReportCompletedCount = draftReportCompletedCount;
+    }
+
+    // final report
+    public void finalReportPendingCount(int finalReportPendingCount) {
+      this.finalReportPendingCount = finalReportPendingCount;
+    }
+
+    public void finalReportCompletedCount(int finalReportCompletedCount) {
+      this.finalReportCompletedCount = finalReportCompletedCount;
+    }
+
+
+
+    public void addCounts(ProjectSummary.Builder builder) {
+      this.totalTestCount += builder.totalTestCount;
+      // receipt
+      this.receiptPendingQcCount += builder.receiptPendingQcCount;
+      this.receiptCompletedCount += builder.receiptCompletedCount;
+      // extraction
+      this.extractionPendingCount += builder.extractionPendingCount;
+      this.extractionPendingQcCount += builder.extractionPendingQcCount;
+      this.extractionCompletedCount += builder.extractionCompletedCount;
+      // library preparation
+      this.libraryPrepPendingCount += builder.libraryPrepPendingCount;
+      this.libraryPrepPendingQcCount += builder.libraryPrepPendingQcCount;
+      this.libraryPrepCompletedCount += builder.libraryPrepCompletedCount;
+      // library qualification
+      this.libraryQualPendingCount += builder.libraryQualPendingCount;
+      this.libraryQualPendingQcCount += builder.libraryQualPendingQcCount;
+      this.libraryQualCompletedCount += builder.libraryQualCompletedCount;
+      // full depth sequencing
+      this.fullDepthSeqPendingCount += builder.fullDepthSeqPendingCount;
+      this.fullDepthSeqPendingQcCount += builder.fullDepthSeqPendingQcCount;
+      this.fullDepthSeqCompletedCount += builder.fullDepthSeqCompletedCount;
+      // informatics review
+      this.informaticsPendingCount += builder.informaticsPendingCount;
+      this.informaticsCompletedCount += builder.informaticsCompletedCount;
+      // draft report
+      this.draftReportPendingCount += builder.draftReportPendingCount;
+      this.draftReportCompletedCount += builder.draftReportCompletedCount;
+      // final report
+      this.finalReportPendingCount += builder.finalReportPendingCount;
+      this.finalReportCompletedCount += builder.finalReportCompletedCount;
+    }
+
+    public ProjectSummary build() {
+      return new ProjectSummary(this);
```

Comment:
`Builder.build()` and `new ProjectSummary(builder)` are both public, though neither seems to ever be called? Which is preferred?